### PR TITLE
Update about page

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
- [x] So far this PR has the completed styles and static markup to make up the new page. However, I'm not sure about which (if any) components should be extracted. I just wrote the styles in the page stylesheet because don't really know if they'll be reused again in other pages. 

- [x] Another pending issue is the data part. I have yet to update the team section to have an abstract, so far there's just a placeholder on the page with static text. Also, should the OSS projects come from some sort of data source as the team members? I just hardcoded them for now.

screencapture-localhost-8000-about-2019-04-11-10_48_59

### CODE REVIEW

File: src/components/team-list/team-list.css [hunk 1]
```diff
@@ -3,39 +3,75 @@
...
   text-decoration: none;
   font-family: var(--font-sans-serif);
-  font-size: var(--size-med-lg);
-  font-weight: 600;
-  letter-spacing: 0.05em;
-  /* scaled from the current scale */
-  font-size: calc(1em / var(--size-scale));
+  font-size: var(--size-large);
+  font-weight: 300;
+  color: var(--color-dark-blue);
```
Comment (cherewaty) [inline src/components/team-list/team-list.css]: We'll need something more contrast-y in dark mode for the text that's dark blue.

Screen Shot 2019-04-22 at 6 22 16 PM

Maybe those items are just always `--color-light-blue`?

Comment (jorgelainfiesta) [reply src/components/team-list/team-list.css]: Thanks! I updated the dark mode styles now.

Comment (taras) [reply src/components/team-list/team-list.css]: Looks much nicer!

Comment (cherewaty) [reply src/components/team-list/team-list.css]: 👏 Nice!

**Comment by cherewaty** [comment]
These are colored as if they're links 🤔 
Screen Shot 2019-04-30 at 1 30 44 PM

Do you think just font weight is enough to get the contrast you're looking for?
Screen Shot 2019-04-30 at 1 30 36 PM

**Comment by jorgelainfiesta** [comment]
I'm not happy at all with the dark styles, to be honest. I think we should reconsider the theme and develop a better color scheme for it. Not sure if I should prioritize that now?

**Comment by taras** [comment]
Let’s finish the content changes first and we’ll look at style changes after. Let’s just keep it passable for now.

**Comment by taras** [comment]
@jorgelainfiesta are we ready to merge this?

**Pull request merged**